src/core/server/saved_objects/saved_objects_config.ts

```diff
@@ -49,12 +49,24 @@ export const savedObjectsConfig = {
   schema: schema.object({
     maxImportPayloadBytes: schema.byteSize({ defaultValue: 26214400 }),
     maxImportExportSize: schema.byteSize({ defaultValue: 10000 }),
+    db: schema.object({
+      type: schema.string({ defaultValue: 'opensearch' }),
```

Comment:
Currently I see two data types are supported in the `selectRepository` method: `opensearch` and `postgres`.  Do you think we should have db type validation for the schema? For example: if user define `db.type = DynamoDB`, schema validation should fail with an exception: `DynamoDB db type is not supported` to inform user the invalid configuration setting for metastorage.

Reply:
Good idea. We can do validation when we productionize code. I am adding [task](https://app.asana.com/0/0/1204142706911754/f) in Asana to track.